### DISCUSSION TITLE
Chart type for seeing request timing spread

### DISCUSSION THREAD

**Discussion by kristofferlindvall** [datalust/seq-tickets#1080] [Ideas]
When I analyse request times, I tend to either look at 95th percentile, max or mean depending on what I'm looking for. But, I was hanging out in Fastly's real-time stats today and figured what I really want to draw (in seq) is something like this:

![image](https://user-images.githubusercontent.com/694537/101929636-6bf29700-3bd7-11eb-9533-32b99eb3a8f0.png)

**Comment by nblumhardt** [comment]
These are great - would love to get some support in :+1:
